src/main/kotlin/org/jitsi/nlj/transform/node/incoming/BitrateCalculator.kt

```diff
@@ -93,7 +93,11 @@ open class BitrateCalculator(
      * Keep track of whether the stream is active (has packets at at least [activePacketRateThreshold])
      */
     val active: Boolean
-        get() = Duration.between(start, clock.instant()) <= GRACE_PERIOD || packetRatePps >= activePacketRateThreshold
+        get() {
+            val threshold = if (Duration.between(start, clock.instant()) <= GRACE_PERIOD) 0
```

Comment:
Isn't the rate still 0 if nothing's been received?

Because this is returned as a `long` we probably don't want to do `threshold = 1` - maybe check whether the bitrate > 0?